packages/manager/apps/user-account/src/index.scss

```diff
@@ -0,0 +1,7 @@
+.user-account {
```

Comment:
its good idea to move these to module

Reply:
I need this in app as bootstrap is already included in dedicated

Reply:
Has been retained in `apps` as it is only required for `standalone` module